tests/test_problem_bank_helpers_rounding.py

```diff
@@ -133,12 +133,25 @@ def test_sigfigs(test_group, variables):
             pytest.fail(f"test group is not defined (got: '{test_group}')")
         assert (pbh.sigfigs(test_input) == correct_sigfigs), f"input: {test_input}, output: {pbh.sigfigs(test_input)}"
 
-def test_sigfigs_floatinput_fail():
+def test_sigfigs_float_input_fail():
     with pytest.raises(Exception):
         pbh.sigfigs(float(1))
 
 
 # Test round_sig function
+def test_roundsig_int_returns_int():
+    """Test rounding an int with specified sigfigs"""
+    assert type(pbh.round_sig(123, 2)) is int
```

Comment:
I personally think `isinstance` is better than checking the identity of the type
```suggestion
    assert isinstace(pbh.round_sig(123, 2), int)
```

Reply:
Yep, I agree the change has been made.